src/status_im2/subs/communities.cljs

```diff
@@ -238,10 +239,23 @@
                  :unread-messages? (pos? unviewed-messages-count)
                  :mentions-count   (or unviewed-mentions-count 0)
                  :locked?          (or (not joined) (not can-post?))
+                 :muted?           muted
                  :id               id})))
     {}
     chats)))
 
+(re-frame/reg-sub
+ :communities/channel-by-id
+ (fn [[_ {:keys [community-id channel-id]}]]
+   [(re-frame/subscribe [:communities/categorized-channels community-id])
+    (atom channel-id)])
+ (fn [[channels channel-id]]
```

Comment:
you have `channel-id` here , no need to wrap it in atom

Reply:
`[channels _ [{:keys [channel-id]}]]`

Reply:
Okay